swagger.yaml

```diff
@@ -5167,6 +5168,8 @@ definitions:
         description: Determine the disk's unique identifier which is the by-id field if it exists and fallback to the by-path field otherwise
       drive_type:
         type: string
+      has_uuid:
+        type: boolean
```

Comment:
I thought that we want to add "unknown" option

Reply:
no need,  it's on the disk level